client/client-multi/client-cpp/toolchains/edison.cmake

```diff
@@ -23,19 +23,23 @@ set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION 3.10.17-poky-edison+)
 set(CMAKE_SYSTEM_PROCESSOR i686)
 
-if(NOT DEFINED EDISON_SDK_ROOT AND WIN32)
-	if(${SDK_32bits})
-		set(EDISON_SDK_ROOT C:/Edison/poky-edison-eglibc-i686-edison-image-core2-32-toolchain-1.7.2)
-	else()
-		set(EDISON_SDK_ROOT C:/Edison/poky-edison-eglibc-x86_64-edison-image-core2-32-toolchain-1.7.2)
-	endif()
-else(NOT DEFINED EDISON_SDK_ROOT)
-	set(EDISON_SDK_ROOT /opt/poky-edison/1.6)
-endif()
+if(NOT DEFINED EDISON_SDK_ROOT)
+    if(WIN32)
+        if(${SDK_32bits})
+            set(EDISON_SDK_ROOT C:/Edison/poky-edison-eglibc-i686-edison-image-core2-32-toolchain-1.7.2)
```

Comment:
This should be better done with use of Environment variables to allow users to change the place.